test/e2e/suite/issuers/acme/certificate/dns01.go

```diff
@@ -1,190 +0,0 @@
-		})
-
-		It("should obtain a signed certificate for a regular domain", func() {
-			By("Creating a Certificate")
-
-			certClient := f.CertManagerClientSet.CertmanagerV1().Certificates(f.Namespace.Name)
-
-			cert := gen.Certificate(certificateName,
-				gen.SetCertificateSecretName(certificateSecretName),
-				gen.SetCertificateIssuer(cmmeta.ObjectReference{Name: issuerName}),
-				gen.SetCertificateDNSNames(dnsDomain),
-			)
-			cert.Namespace = f.Namespace.Name
-
-			cert, err := certClient.Create(context.TODO(), cert, metav1.CreateOptions{})
-			Expect(err).NotTo(HaveOccurred())
-
-			By("Waiting for the Certificate to be issued...")
-			err = f.Helper().WaitCertificateIssued(f.Namespace.Name, certificateName, time.Minute*5)
-			Expect(err).NotTo(HaveOccurred())
-
-			By("Validating the issued Certificate...")
-			err = f.Helper().ValidateCertificate(f.Namespace.Name, certificateName, validations...)
-			Expect(err).NotTo(HaveOccurred())
-		})
-
-		It("should obtain a signed certificate for a wildcard domain", func() {
-			By("Creating a Certificate")
-
-			cert := gen.Certificate(certificateName,
-				gen.SetCertificateSecretName(certificateSecretName),
-				gen.SetCertificateIssuer(cmmeta.ObjectReference{Name: issuerName}),
-				gen.SetCertificateDNSNames("*."+dnsDomain),
-			)
-			cert.Namespace = f.Namespace.Name
-
-			cert, err := f.CertManagerClientSet.CertmanagerV1().Certificates(f.Namespace.Name).Create(context.TODO(), cert, metav1.CreateOptions{})
-			Expect(err).NotTo(HaveOccurred())
-
-			By("Waiting for the Certificate to be issued...")
-			err = f.Helper().WaitCertificateIssued(f.Namespace.Name, certificateName, time.Minute*5)
-			Expect(err).NotTo(HaveOccurred())
-
-			By("Validating the issued Certificate...")
-			err = f.Helper().ValidateCertificate(f.Namespace.Name, certificateName, validations...)
-			Expect(err).NotTo(HaveOccurred())
-		})
-
-		It("should obtain a signed certificate for a wildcard and apex domain", func() {
-			By("Creating a Certificate")
-
-			cert := gen.Certificate(certificateName,
-				gen.SetCertificateSecretName(certificateSecretName),
-				gen.SetCertificateIssuer(cmmeta.ObjectReference{Name: issuerName}),
-				gen.SetCertificateDNSNames("*."+dnsDomain, dnsDomain),
-			)
-			cert.Namespace = f.Namespace.Name
-
-			cert, err := f.CertManagerClientSet.CertmanagerV1().Certificates(f.Namespace.Name).Create(context.TODO(), cert, metav1.CreateOptions{})
-			Expect(err).NotTo(HaveOccurred())
-			// use a longer timeout for this, as it requires performing 2 dns validations in serial
-			By("Waiting for the Certificate to be issued...")
-			err = f.Helper().WaitCertificateIssued(f.Namespace.Name, certificateName, time.Minute*10)
-			Expect(err).NotTo(HaveOccurred())
-
-			By("Validating the issued Certificate...")
-			err = f.Helper().ValidateCertificate(f.Namespace.Name, certificateName, validations...)
-			Expect(err).NotTo(HaveOccurred())
-		})
-	})
-}
```

Comment:
I can't find where this went 😅 @wallrj

Reply:
Some references:
https://github.com/jetstack/cert-manager/blob/c7514d92624613395eb00da8594ffa5391a0b169/test/e2e/suite/conformance/certificates/tests.go#L58-L81
https://github.com/jetstack/cert-manager/blob/c7514d92624613395eb00da8594ffa5391a0b169/test/e2e/suite/conformance/certificates/tests.go#L427-L450
https://github.com/jetstack/cert-manager/blob/c7514d92624613395eb00da8594ffa5391a0b169/test/e2e/suite/conformance/certificates/tests.go#L452-L476